litestar/contrib/jinja.py

```diff
@@ -70,3 +81,15 @@ def register_template_callable(self, key: str, template_callable: Callable[[dict
             None
         """
         self.engine.globals[key] = pass_context(template_callable)
+
+    @classmethod
+    def from_environment(cls, jinja2_environment: Environment) -> JinjaTemplateEngine:
```

Comment:
I wouldn't use jinja2, simply jinja -same everywhere

Reply:
done in https://github.com/litestar-org/litestar/pull/2195/commits/f3a47fc33f128e972b68c620ba3e26665cc2738a